docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -177,9 +177,15 @@ For more information, see xref:security-openid-connect-oidc-configuration-proper
 
 === Start and configure the Keycloak server
 
-NOTE: Do not start the Keycloak server when you run the application in a dev mode - `Dev Services for Keycloak` will launch a container. See <<keycloak-dev-mode, Running the Application in Dev mode>> section below for more information. Make sure to put the {quickstarts-tree-url}/security-openid-connect-quickstart/config/quarkus-realm.json[realm configuration file] on the classpath (`target/classes` directory) so that it gets imported automatically when running in dev mode - unless you have already built a {quickstarts-tree-url}/security-openid-connect-quickstart[complete solution] in which case this realm file will be added to the classpath during the build.
+[NOTE]
+====
+Do not start the Keycloak server when you run the application in a dev mode - `Dev Services for Keycloak` will launch a container.
+See the <<keycloak-dev-mode, Running the Application in Dev mode>> section below for more information.
```

Comment:
@MichalMaler: Think this link update was missed. Update to 
xref:keycloak-dev-model[Run the application in Dev mode ]

Reply:
Just adding the context. This was not a scope of this PR, and, it works as I was sharing on slack and saying during our two recent calls:
![image](https://user-images.githubusercontent.com/48474054/224276901-39ada279-a99b-42f2-9dd2-691b9cf935f2.png)
Anchor was fix in my previous PR. Render properly in downstream too.

Reply:
But I can add this for the sake of consistency :)

Reply:
CC: @inoxx03 
Hello Stefan, this is the example of what is working on P2 (as our today's call discussion)